src/redturtle/prenotazioni/restapi/services/bookings/search.py

```diff
@@ -87,6 +95,9 @@ def query(self):
         if review_state:
             query["review_state"] = review_state
 
+        if len(query) <= empty_query_len:
+            raise BadRequest(_("No search parameters were passed."))
```

Comment:
if you want to translate this message, you have to wrap it into a api.portal.translate:

`api.portal.translate(_("No search parameters were passed."))`

and i don't see updated *po files

Reply:
It doesnt need translation right now but the code is adaptable for translation if needed in future

Reply:
done

Reply:
why

Reply:
fatto